taxonomy-editor-frontend/src/pages/search/results.jsx

```diff
@@ -117,17 +127,17 @@ const SearchResults = ({query}) => {
                         </TableRow>
                         </TableHead>
                         <TableBody>
-                            {nodes.map((node) => (
+                            {nodes.map((nodeID) => (
```

Comment:
nit: should `nodes` be `nodeIds`   ... posibly `nodeID` be `nodeId` (just to be consistent in the upper/lower casing)

Reply:
Yep, the above naming scheme has been implemented.